nixos/modules/services/web-apps/matomo.nix

```diff
@@ -14,16 +14,53 @@ let
 
   fqdn =
     let
-      join = hostName: domain: hostName + optionalString (domain != null) ".${domain}";
-     in join config.networking.hostName config.networking.domain;
+      join = hostName: domain:
+        hostName + optionalString (domain != null) ".${domain}";
+    in
+    join config.networking.hostName config.networking.domain;
 
-in {
+in
+{
   imports = [
-    (mkRenamedOptionModule [ "services" "piwik" "enable" ] [ "services" "matomo" "enable" ])
-    (mkRenamedOptionModule [ "services" "piwik" "webServerUser" ] [ "services" "matomo" "webServerUser" ])
-    (mkRemovedOptionModule [ "services" "piwik" "phpfpmProcessManagerConfig" ] "Use services.phpfpm.pools.<name>.settings")
-    (mkRemovedOptionModule [ "services" "matomo" "phpfpmProcessManagerConfig" ] "Use services.phpfpm.pools.<name>.settings")
-    (mkRenamedOptionModule [ "services" "piwik" "nginx" ] [ "services" "matomo" "nginx" ])
+    (mkRenamedOptionModule [
+      "services"
+      "piwik"
+      "enable"
+    ] [
+      "services"
+      "matomo"
+      "enable"
+    ])
+    (mkRenamedOptionModule [
+      "services"
+      "piwik"
+      "webServerUser"
+    ] [
+      "services"
+      "matomo"
+      "webServerUser"
+    ])
+    (mkRemovedOptionModule [
+      "services"
+      "piwik"
+      "phpfpmProcessManagerConfig"
+    ]
+      "Use services.phpfpm.pools.<name>.settings")
+    (mkRemovedOptionModule [
+      "services"
+      "matomo"
+      "phpfpmProcessManagerConfig"
+    ]
+      "Use services.phpfpm.pools.<name>.settings")
+    (mkRenamedOptionModule [
+      "services"
+      "piwik"
+      "nginx"
+    ] [
+      "services"
+      "matomo"
+      "nginx"
+    ])
```

Comment:
I can't emphasize enough how in my opinion the formatting in this block is **not** an improvement :confused: 

To be clear, though, the rest of the file is fine... again, in **my** opinion :smile:

Reply:
that one specifically is what I had in mind when I made my comment about lists. :smile:  I think nixpkgs-fmt did half of them and it was somehow worse so I did the rest so it wouldn't keep messing with it ( and that's how lists are supposed to be anyway but the reasoning probably doesn't matter here so :| ) I switched nixops-digitalocean to nixfmt since [lol](https://github.com/Kiwi/lol-nix-formatter) I found that one by accident there are a lot more in nixpkgs that it will rewrite every time. ...which seems problematic if it's meant to be put in git hooks? idk

ANYWAY I'll do something about that one and I'll see about release notes too. then I can check the documentation box for once.